shared/common-adapters/hoc-timers.js

```diff
@@ -17,8 +17,13 @@ function clearId(clearFunc: (id?: number) => void, array: Array<number>, id?: ?n
   }
 }
 
+function getDisplayName(WrappedComponent) {
```

Comment:
this passes through the displayname so we can tell what this is. was super confusing in the react view